### PR TITLE
Autogenerate introspection data

### PR DESCRIPTION
**Pull request by mulkieran**
Related: https://github.com/stratis-storage/project/issues/225

### CODE REVIEW

**Comment by mulkieran** [comment]
Blocked by https://github.com/stratis-storage/stratisd/pull/2228.

**Comment by mulkieran** [comment]
Doesn't close the issue; after this is merged, I should also fix up the bit in the D-Bus tests in stratisd, which do still exist as they're used by the udev tests.

**Review by jbaublitz — APPROVED** [review]

**Pull request merged**
